frontend/src/routes/index.js

```diff
@@ -12,28 +11,31 @@ import NewParticipantView from "../views/NewParticipantView"
 const Routes = () => {
   return (
     <BrowserRouter>
-      <Navbar />
-      <Route path="/login" component={LoginForm} />
-      <MainLayout>
```

Comment:
I also added an additional Switch and Router around MainLayout so that the MainLayout component isn't on the page at the same time as the LoginForm Component.